### PR TITLE
Document a simpler xvfb setup

### PR DESCRIPTION
**Pull request by chadwhitacre**
#1060 redux.

- point out `xvfb-run`
- fix a typo while we're at it

### CODE REVIEW

File: user/gui-and-headless-browsers.md [hunk 1]
```diff
@@ -53,9 +53,26 @@ For travis-web, our very own website, we use Sauce Labs to run browser tests on
 To run tests requiring a graphical user interface on Travis CI, use `xvfb` (X
 Virtual Framebuffer) to imitate a display. If you need a browser, you can use
 Firefox (either with the pre-installed version, or the [addon](/user/firefox))
-or Google Chorme (with the [addon](/user/chrome), on Linux Trusty or OS X).
+or Google Chrome (with the [addon](/user/chrome), on Linux Trusty or OS X).
 
-Start `xvfb` in the `before_script` section of your `.travis.yml`:
+### Simple
```
Comment (BanzaiMan) [inline user/gui-and-headless-browsers.md]: I feel that "Simple" vs "Complex" is not a helpful distinction here.

File: user/gui-and-headless-browsers.md [hunk 2]
```diff
@@ -53,9 +53,26 @@ For travis-web, our very own website, we use Sauce Labs to run browser tests on
...
+
+For simple cases, try wrapping your `script` call with `xvfb-run`, like so:
```
Comment (BanzaiMan) [inline user/gui-and-headless-browsers.md]: "Wrapping" is not the right word. The default `script` phase is not wrapped, but overridden.

File: user/gui-and-headless-browsers.md [hunk 3]
```diff
@@ -53,9 +53,26 @@ For travis-web, our very own website, we use Sauce Labs to run browser tests on
...
+
+```yaml
+script: xvfb-run make test
+```
+
+To set the screen resolution, try:
```
Comment (BanzaiMan) [inline user/gui-and-headless-browsers.md]: Strike "try".

File: user/gui-and-headless-browsers.md [hunk 4]
```diff
@@ -53,9 +53,26 @@ For travis-web, our very own website, we use Sauce Labs to run browser tests on
...
+
+```yaml
+script: xvfb-run --server-args="-screen 0 1024x768x24" make test
+```
+
+### Complex
+
+If you can't use `xvfb-run` for some reason, try calling `xvfb` itself, in the
```
Comment (BanzaiMan) [inline user/gui-and-headless-browsers.md]: "for some reason" invites the question "What *are* such reasons?" I don't think we are actually addressing that, so I think it is best to not mention that.

**Comment by whit537** [comment]
Thanks, @BanzaiMan. Didn't we meet at the Community Leadership Summit a few weeks ago? :)

I've attempted to address your review comments in 7e9c9e2545fd1362d6c914a5d2e3673c4c4dca3e. Please let me know how that looks.

File: user/gui-and-headless-browsers.md [hunk 5]
```diff
@@ -53,9 +53,27 @@ For travis-web, our very own website, we use Sauce Labs to run browser tests on
 To run tests requiring a graphical user interface on Travis CI, use `xvfb` (X
 Virtual Framebuffer) to imitate a display. If you need a browser, you can use
 Firefox (either with the pre-installed version, or the [addon](/user/firefox))
-or Google Chorme (with the [addon](/user/chrome), on Linux Trusty or OS X).
+or Google Chrome (with the [addon](/user/chrome), on Linux Trusty or OS X).
 
-Start `xvfb` in the `before_script` section of your `.travis.yml`:
+### Less Fuss
```
Comment (plaindocs) [inline user/gui-and-headless-browsers.md]: This is great, but could you rename the Less/More fuss headings to something like

# Using the xvfb-run wrapper

# Using xvfb directly

Cheers

**Comment by plaindocs** [comment]
Also related to #1206

**Comment by chadwhitacre** [comment]
Closing in favor of #1696 (I forgot about this PR and deleted my old fork :).

**Pull request closed without merging**